assets/cubyz/recipes/brick_recipes.zig.zon

```diff
@@ -1,114 +1,18 @@
 .{
 	.{
-		.inputs = .{"cubyz:marble"},
-		.output = "cubyz:marble_tile",
+		.inputs = .{"{item}.stone"},
```

Comment:
I'm still not convinced that we need tags.
For now I think we can afford to just craft everything into tiles if present. And in the future I think we should have a `stone/` folder. The current file way the files are ordered is really difficult to work with and a hierarchical would be beneficial in many ways.

Are there any cases where you do think a tag would be needed, even after the assets are ordered in a better way?